workflow/rocoto_funcs/prep_lbc.py

```diff
@@ -51,11 +51,18 @@ def prep_lbc(xmlFile, expdir, do_ensemble=False):
   starttime=get_cascade_env(f"STARTTIME_{task_id}".upper())
   if realtime.upper() == "TRUE":
     timedep=f'\n   <timedep><cyclestr offset="{starttime}">@Y@m@d@H@M00</cyclestr></timedep>'
+
+  taskdep=""
+  for hr in range(0,12):
+    taskdep=taskdep + f'\n <metataskdep metatask="lbc{ensindexstr}" cycle_offset="-{hr}:00:00" />'
   
   dependencies=f'''
   <dependency>
   <and>{timedep}
    <taskdep task="prep_ic{ensindexstr}"/>
+   <or>
```

Comment:
Line 62: not introduced by this PR, but curious why prep_lbc depends on prep_ic?

Reply:
@guoqing-noaa Do you want answer to this question before it is merged?

Reply:
This was introduced by Ming so he knows the best logic behind this dependency, although the prep_lbc doesn't seem to need this, especially after adding the task depencies on the lbc tasks.